packages/backend/src/telemetry/transaction-amount.ts

```diff
@@ -0,0 +1,53 @@
+import { ValueType } from '@opentelemetry/api'
+import { ConvertError, RatesService } from '../rates/service'
+import { ConvertOptions } from '../rates/util'
+import { TelemetryService } from './service'
+import { privacy } from './privacy'
+import { Asset } from '../rates/util'
+
+export async function collectTelemetryAmount(
+  telemetryService: TelemetryService,
+  aseRates: RatesService,
+  { amount, asset }: { amount: bigint; asset: Asset }
+) {
+  if (!amount) {
+    return
+  }
+  const convertOptions = {
+    sourceAmount: amount,
+    sourceAsset: { code: asset.code, scale: asset.scale },
+    destinationAsset: {
+      code: telemetryService.getBaseAssetCode(),
+      scale: telemetryService.getBaseScale()
+    }
+  }
+  const converted = await convertAmount(
+    aseRates,
+    telemetryService.getRatesService(),
+    convertOptions
+  )
+  if (converted === ConvertError.InvalidDestinationPrice) {
```

Comment:
log here maybe as well

Reply:
I'm waiting on the previous comment resolution, so i know where to inject the app context logger for it

Reply:
moved the covnersion inside of telemetry service.
It now logs using telemetry's logger in case both ase rates service and fallback rates service conversions fail.
This just remains an early return